test/torch/tensors/test_native.py

```diff
@@ -205,3 +205,24 @@ def forward(self, x):
 
     ## Forward on the remote model
     pred = model_net(tensor_remote)
+
+
+def test_encrypt():
+    hook = syft.TorchHook(torch)
```

Comment:
Check other examples you can improt workers directly:

ex: in test_additive_sharing.py
```
def test_eq(workers):
    me, alice, bob, crypto_provider = (
        workers["me"],
        workers["alice"],
        workers["bob"],
        workers["james"],
    )
```